test/e2e/Deploy.t.sol

```diff
@@ -432,49 +432,29 @@ abstract contract CommonDeploymentTest is HaiTest, Deploy {
   function test_TokenDistributor_Params() public {
     assertEq(tokenDistributor.root(), _tokenDistributorParams.root);
     assertEq(tokenDistributor.totalClaimable(), _tokenDistributorParams.totalClaimable);
-    assertEq(tokenDistributor.claimPeriodStart(), _tokenDistributorParams.claimPeriodStart);
+    // NOTE: (deployment)block.timestamp + 1 days (cannot be tested)
+    // assertEq(tokenDistributor.claimPeriodStart(), _tokenDistributorParams.claimPeriodStart);
```

Comment:
the pain about that, is that when that date is on the past, the deployment script breaks, so preferred to use `block.timestamp + x` in testnet, instead of changing the date every time, for mainnet we already have a defined date, and is good it reverts, but testnet is pain